ibm/service/cos/resource_ibm_cos_bucket.go

```diff
@@ -95,10 +95,18 @@ func ResourceIBMCOSBucket() *schema.Resource {
 				Description: "CRN of resource instance",
 			},
 			"key_protect": {
-				Type:        schema.TypeString,
-				ForceNew:    true,
-				Optional:    true,
-				Description: "CRN of the key you want to use data at rest encryption",
+				Type:          schema.TypeString,
+				ForceNew:      true,
+				Optional:      true,
```

Comment:
Do we need to mark this as deprecated

Reply:
As of now we are not deprecating the key_protect parameter.
we will just be recommending new customers to use the new parameter.